test/fixtures/output/custom-code-logging/create-no-env.golden

```diff
@@ -1,6 +1,9 @@
-+-------------+------------+
-| Id          | ccl-123456 |
-| Cloud       | AWS        |
-| Region      | us-west-2  |
-| Environment | env-000000 |
-+-------------+------------+
++-------------+-------------+
+| Id          | ccl-123456  |
+| Cloud       | AWS         |
+| Region      | us-west-2   |
+| Environment | env-000000  |
+| Topic       | topic-123   |
+| Cluster     | cluster-123 |
```

Comment:
Does cluster refer to a Kafka cluster? Normally that would start with "lkc-".

Reply:
yes, it's kafka cluster. do you suggest to change this name?